backend/api/newsletters/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+
+from newsletters.models import Subscription
+from strawberry_forms.forms import FormWithContext
+
+
+class SubscribeToNewsletterForm(FormWithContext):
+    email = forms.EmailField()
+
+    def save(self):
+        email = self.cleaned_data.get("email")
+        subscription = Subscription.objects.get_or_create(email=email)[0]
+        return subscription
+
+
+class UnsubscribeToNewsletterForm(FormWithContext):
+    email = forms.EmailField()
+
+    def save(self):
+        email = self.cleaned_data.get("email")
+        try:
+            deleted, _ = Subscription.objects.get(email=email).delete()
+            return deleted == 1
```

Comment:
we can just have this, `Subscription.objects.get(email=email).delete()` no need to check :)